src/rollover.py

```diff
@@ -36,6 +37,36 @@ def _populate_ecs_info(self):
         info = self.ecs_client.describe_instances([self.ecs_id])
         self.ec2_id = info[self.ecs_id]['ec2InstanceId']
 
+        # store resource utilization
+        self.resources = {
+            "cpu": {
+                "registered": -1,
+                "remaining": -1,
+                "utilized": -1,
+            },
+            "memory": {
+                "registered": -1,
+                "remaining": -1,
+                "utilized": -1,
+            },
+        }
+
```

Comment:
changed to use those var names and added docstring. https://github.com/Clever/ecs-rollover/pull/7/commits/8128e4b90b67c554ba481e1d2c4b96575b8f5625